src/index.html

```diff
@@ -17,6 +17,77 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'Logo', provide a more descriptive alternative text that describes the image, such as the company name or logo description.

Reply:
Changed the attribute alt, add name company